codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/EndpointHostPrefixMiddleware.java

```diff
@@ -83,7 +84,7 @@ public void writeAdditionalFiles(
                         middlewareHelperName,
                         () -> {
                             writer.write(
-                                    "return stack.Serialize.Insert(&$L{}, `OperationSerializer`, middleware.Before)",
+                                    "return stack.Serialize.Insert(\"OperationSerializer\", middleware.Before, &$L{})",
```

Comment:
This was mainly to bias towards the generated output Go code consistency around double quote usage rather then convenience to the Java side. Otherwise can switch it back if we prefer that.